demos/1 - download_microns_digital_twin.ipynb

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
<p><code>params_core.pt</code></p><p> </p>

<pre class="ql-syntax" spellcheck="false">A PyTorch state dictionary file containing the foundation core model parameters. 

This file includes only parameters whose keys start with "core." from the original model state dictionary. 

It is generated once and shared across all scan instances.
</pre><p>It is generated once and shared across all scan instances. -&gt; The core parameters are learned by training on the foundation cohort scans. Once learned, the core parameters are frozen and transferred to all other scans.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cajal/fnn/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>